templates/aws-stack.yml

```diff
@@ -626,10 +632,13 @@ Rules:
             - !Equals
               - !Ref PipelineSigningKMSKeyId
               - ""
+            - !Equals
+              - !Ref PipelineSigningKMSKeyARN
+              - ""
             - !Equals
```

Comment:
Should we add a separate rule to express that you shouldn't provide PipelineSigningKMSKeyId and PipelineSigningKMSKeyArn at the same time?

Reply:
Done